README.md

```diff
@@ -33,9 +46,15 @@ make deps
 make run
 ```
 
-## Local (Development)
+> You can use the `make watch` command to run the application on watching mode, allowing it to be restarted automatically when the code changes.
+
+## Docker installation and usage
+
+### Requirements
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-duplicate-headings] Do not use headings with similar content (18:1)](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4651373)